### PR TITLE
gulpfile.babel: fix the issue that only build on macOS

### PR DESCRIPTION
**Pull request by Ephemera**
### Description
I use `gulp-crx-pack` library and fake private key for chrome extension packaging.
If you have any ideas about private key or key location, tell me, please.

### CODE REVIEW

File: gulpfile.babel.js [hunk 1]
```diff
@@ -80,15 +80,15 @@ gulp.task('chrome:zip', () => {
...
-  $.run('"/Applications/Google Chrome.app/Contents/MacOS/Google Chrome"' +
-    ' --pack-extension=' + path.join(__dirname, './tmp/chrome') +
-    ' --pack-extension-key=' + path.join(process.env.HOME, '.ssh/chrome.pem')
-  ).exec(cb)
-})
-
-gulp.task('chrome:crx', ['chrome:_crx'], () => {
-  return pipe('./tmp/chrome.crx', './dist')
+gulp.task('chrome:crx', () => {
+  const real = '.ssh/chrome.pem'
```
Comment (buunguyen) [inline gulpfile.babel.js]: Should be `path.join(process.env.HOME, '.ssh/chrome.pem')`

File: test/fake.pem [hunk 2]
```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```
Comment (buunguyen) [inline test/fake.pem]: Suggest move this file to root directory instead of `test`, rename to `test_chrome_key.pem`

File: gulpfile.babel.js (hunk 1 above)
Comment (buunguyen) [reply gulpfile.babel.js]: This is better :) path.join(os.homedir(), '.ssh/chrome.pem')

**Comment by buunguyen** [comment]
Sorry for the late response, been traveling. Just some minor suggestions.

**Comment by Ephemera** [comment]
@buunguyen 

It's okay. Was your travel good? 
I fixed your suggestions.

**Comment by buunguyen** [comment]
Thanks for taking long. Just merged!

**Pull request merged**